### PR TITLE
Add fault injection examples

### LINKED ISSUE #583: Add fault injection examples
## User stories
As a user I want to have examples on how to configure the fault injection in order to use the fault injection by myself.

## Acceptance Criteria
- [ ] Add examples from Branch [FaultInjection-examples](https://github.com/BP2022-AP1/bp2022-ap1/tree/FaultInjection-examples)
- [ ] add docstrings

### PR DESCRIPTION
**Pull request by instant-sky**
Fixes #583 

Adds examples for the configuration of the fault injection.

## PR checklist

- [x] Acceptance criteria fulfilled
- [ ] ~Additional features are tested~
- [x] Docs (code, wiki & diagrams) updated
- [ ] ~Breaking changes anounced~
- [ ] ~Added classes that inherit from `BaseModel` to `src.constants.tables`~
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [x] Another dev reviewed and approved

### CODE REVIEW

File: scripts/insert_fault_injection_examples.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+    # --- different schedules to try out the fault injection --- #
+    regular_schedule = ScheduleConfiguration.create(
+        schedule_type="TrainSchedule",
+        strategy_type="RegularScheduleStrategy",
+        strategy_start_time=0,
+        strategy_end_time=7200,
+        train_schedule_train_type="regio",
+        regular_strategy_frequency=180,
+        regular_strategy_frequency=400,
```
Comment (antonykamp) [inline scripts/insert_fault_injection_examples.py]: You listed `regular_strategy_frequency`two times 😇

**Review by antonykamp — COMMENTED** [review]
I found only a small typo 😇

File: scripts/insert_fault_injection_examples.py (hunk 1 above)
Comment (instant-sky) [reply scripts/insert_fault_injection_examples.py]: Oh, you are right!
Oh, you are right!
🐝

**Review by antonykamp — APPROVED** [review]
Good work 👍

**Pull request merged**